client/components/forms/form-phone-media-input/index.jsx

```diff
@@ -30,6 +34,8 @@ export default class extends React.Component {
 					<PhoneInput
 						{ ...omit( this.props, [ 'className', 'countryCode' ] ) }
 						ref="input"
+						// eslint-disable-next-line react/jsx-no-bind
+						onRef={ ref => this.phoneInput = ref }
```

Comment:
Also, property name sounds weird. Shouldn't it be something like, `setFocusComponent`.

Reply:
`this.phoneInput` is a reference to the `PhoneInput` component, so I think it seems appropriate. Maybe I'm thinking of it wrong though?

Reply:
I think he refers to the `onRef` prop.

Reply:
How about `setComponentReference`?